queries.sql

```diff
@@ -8,4 +8,44 @@ select date_of_birth from animals where name = 'Augmon' or name = 'Pikachu'
 select name, escape_attempts from animals where weight_kg > 10.5
 select * from animals where neutered = true
 select * from animals where name <> 'Gabumon'
-select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
\ No newline at end of file
+select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
+
+begin;
+update animals set species = 'unspecified';
+rollback;
+
+select * from animals
+
+begin;
+update animals set species = 'digimon' where name like '%mon%';
+update animals set species = 'pokemon' where species is NULL;
+commit;
+
+select species from animals
+
+begin
+delete * from animals
+rollback
+
+begin;
+delete  from animals
+rollback
+
+select * from animals
+
+begin;
+delete from animals where date_of_birth > '2022-01-01';
+savepoint sp1
+update animals set weight_kg = weight_kg * -1
+rollback to sp1
+update animals set weight_kg = weight_kg * -1 where weight_kg < 0;
+commit
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly make sure to verify that changes have been made before and after `COMMIT`. Something like this: 

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit